### PR TITLE
{phys} [intel/2016a] ALPS  (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

File: easybuild/easyconfigs/a/ALPS/ALPS-2.2.b4-intel-2016a-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+configopts = '-DSZIP_INCLUDE_DIR=$EBROOTSZIP/include -DSZIP_LIBRARY=$EBROOTSZIP/lib '
+configopts += '-DFFTW_LIBRARIES=$FFTW_LIB_DIR -DFFTW_INCLUDE_DIR=$FFTW_INC_DIR'
+
+separate_build_dir = True
+
+maxparallel = 1
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ["."]
```
Comment (boegel) [inline easybuild/easyconfigs/a/ALPS/ALPS-2.2.b4-intel-2016a-Python-2.7.11.eb]: :)

Comment (wpoely86) [inline easybuild/easyconfigs/a/ALPS/ALPS-2.2.b4-intel-2016a-Python-2.7.11.eb]: It's wip for a reason :wink:

Doesn't yet work but in this way, I will not forget about it

Comment (boegel) [inline easybuild/easyconfigs/a/ALPS/ALPS-2.2.b4-intel-2016a-Python-2.7.11.eb]: well, the sanity check will pass like this...

in a WIP easyconfig, I keep both lists empty, which will make EB complain

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2685.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9e72e88986f1a1879defb81acd340bc8 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7c72267e79b4d0ce251d66fa8d0a9e24 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**